<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QDialog" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="telas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="pg01_inicial">
    <widget class="QPushButton" name="pg01_bt_carregar_arquivo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Carregar Arquivo</string>
     </property>
    </widget>
    <widget class="QLabel" name="pg01_label_titulo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg02_pos_inicial">
    <widget class="QPushButton" name="pg02_bt_verific_empre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Verificar Empresas</string>
     </property>
    </widget>
    <widget class="QListWidget" name="pg02_lista_empresas">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>256</width>
       <height>141</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Teste1</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="pg02_label_infor_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pg02_bt_iniciar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
